examples/cinematography_domain/client.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import annotations
+
+import logging
+from typing import List, Optional, cast
+
+from cachelib import BaseCache, SimpleCache
+from cinematography_domain.schema import Movie, Person, cine_schema
```

Comment:
Should this be?
```suggestion
from .schema import Movie, Person, cine_schema
```

Reply:
Thing is I have a "main" in this file as well, so this won't work if the file is executed directly.

Reply:
On the second thought, let's create a `__main__.py` and put the usage code in there... 👍